### DISCUSSION TITLE
Phrase list

### DISCUSSION THREAD

**Discussion by OBERMEYER-DD** [chidiwilliams/buzz#901] [Ideas]
A phrase list to improve transcription accuracy by providing a list of known phrases, such as names of people or specific locations.
To seperate each value in the phrase list you could use commas or semicolons.

**Comment by chidiwilliams** [comment]
@OBERMEYER-DD, you can already do this by clicking 'Advanced Settings' in the transcription options.

**Reply by OBERMEYER-DD** [reply]
Which option ist it? Sorry, I'm from germany
![image](https://github.com/user-attachments/assets/500007ef-699c-41c2-a7ca-f918dd1042ac)

**Reply by raivisdejus** [reply]
Please see this https://cookbook.openai.com/examples/whisper_prompting_guide

**Comment by raivisdejus** [comment]
Another option is to use the "Translation feature" https://chidiwilliams.github.io/buzz/docs/usage/translations

Translation feature passes all transcripts to LLM like ChatGPT with some instructions of what it should do with the transcript sentences provided. It can translate it to some language but you can also instruct it to fix spelling and some common names. To get the best results you need, you will have to play around with prompt but I can imagine providing the translation feature with the following prompt:

> You are professional German language linguist that will find and correct small mistakes in the sentences provided to you. Sentences provided to you may contain the following common names - Berlin, BMW, Brandenburg. Fix any possible errors in the sentences provided to you and do not add any other comments or notes, your responses will be used in an automated tool.
